### DISCUSSION TITLE
You have reached the maximum number (10) of Address Books.

### DISCUSSION THREAD

**Discussion by isential-gmbh** [rustdesk/rustdesk#7748] [Q&A] [answered]
![image](https://github.com/rustdesk/rustdesk/assets/68383884/2fde1b71-147a-4f11-aab1-25856d5300a2)

We have acquired the following license:

"RustDesk 10 Users, 200 Devices"

Now, we are getting the error message above. However, we have more than 10 clients that we need to remotely assist. Each client has multiple devices. Therefore, we wanted to create an address book for each client and are now receiving this error message. How should it be understood?

**Comment by 21pages** [comment]
You can add 10 shared address books, each address book can add 200 devices.
[video](https://private-user-images.githubusercontent.com/71636191/320276751-6a73e998-8623-416f-98c8-04316ff66db5.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzOTkwNjEsIm5iZiI6MTcxMzM5ODc2MSwicGF0aCI6Ii83MTYzNjE5MS8zMjAyNzY3NTEtNmE3M2U5OTgtODYyMy00MTZmLTk4YzgtMDQzMTZmZjY2ZGI1Lm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE4VDAwMDYwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzMjgwNTM5YzVhZmFjZTYxZjU3ODgyNjM0NGUxMTE3YzRmNTIwZDZkNzE4N2Y5ODBkY2E3Y2U1NTg2ZDg0MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y7ofLzVkfSXsq50db3YqYQhbMR0x0BtfaRgBB0vHTdM)

**Reply by isential-gmbh** [reply]
What sense do the following limitations make? I don't understand. If I could have an address book for each customer with as many entries as needed, that would be an excellent solution. For example, that's how it works with the well-known product AnyDesk.

Does this limitation relate to the number of users?

**Reply by 21pages** [reply]
~Thank you for your feedback. In the next version, each user will be allowed to create a number of address books equal to the licensed user count. For example, if you have 10 licensed users, each user can create 10 address books, with a total limit of 100 address books for your license.~

**Reply by 21pages** [reply]
@isential-gmbh 

**Update**:  No limit of the number of address books in the next version.

**Comment by rustdesk** [comment, marked as answer]
Thanks for your feedback, we will remove this limit in next version.

**Comment by isential-gmbh** [comment]
Great! Thank you very much! We're looking forward to the new version. How can I find out when this new version is released?
